qcodes/dataset/data_set.py

```diff
@@ -1111,3 +1180,28 @@ def new_data_set(name, exp_id: Optional[int] = None,
                 metadata=metadata, exp_id=exp_id)
 
     return d
+
+
+def generate_dataset_table(guids: Sequence[str],
+                           conn: Optional[ConnectionPlus] = None) -> str:
+    """
+    Generate an ASCII art table of information about the runs attached to the
+    supplied guids.
+
+    Args:
+        guids: Sequence of one or more guids
+        conn: A ConnectionPlus object with a connection to the database.
+
+    Returns: ASCII art table of information about the supplied guids.
+    """
+    from tabulate import tabulate
+    headers = ["captured_run_id", "experiment_name", "sample_name",
+               "sample_id", "location", "work_station"]
```

Comment:
Should `captured_counter` not be one of the headers?

Reply:
Yes that is probably a good idea